apps/builder/app/builder/features/ai/ai-command-bar.tsx

```diff
@@ -147,14 +152,17 @@ export const AiCommandBar = () => {
   let textAreaValue = value;
   let textAreaDisabled = false;
 
-  let recordButtonTooltipContent = "Start recording";
```

Comment:
can we please move all the logic into a function?

Reply:
I will do some refactor when changes will be more rare